src/zui/ZUIEditTextInPlace/index.tsx

```diff
@@ -44,18 +51,28 @@ const useStyles = makeStyles((theme) => ({
 
 export interface ZUIEditTextinPlaceProps {
   allowEmpty?: boolean;
+  autoOnEdit?: boolean;
+  currentType?: string;
```

Comment:
`ZUIEditTextInPlace`, being a general-purpose reusable component, should not know anything about _types_, a very specific aspect of the event feature in Zetkin. When you find yourself adding properties like this, named after a very specific feature, the separation of concerns alarm bells should go off.